libraries/aws/jobs/src/aws_iot_jobs_serialize.c

```diff
@@ -897,108 +897,109 @@ static AwsIotJobsError_t _generateUpdateRequest( const AwsIotJobsRequestInfo_t *
     if( pJobsRequest == NULL )
     {
         IotLogError( "No memory for Jobs UPDATE request." );
-
-        IOT_SET_AND_GOTO_CLEANUP( AWS_IOT_JOBS_NO_MEMORY );
+        status = AWS_IOT_JOBS_NO_MEMORY;
     }
-
-    /* Clear the request JSON. */
-    ( void ) memset( pJobsRequest, 0x00, requestLength );
-
-    /* Construct the request JSON. */
-    APPEND_STRING( pJobsRequest, copyOffset, "{\"", 2 );
-
-    /* Add the status. */
-    APPEND_STRING( pJobsRequest, copyOffset, STATUS_KEY, STATUS_KEY_LENGTH );
-    APPEND_STRING( pJobsRequest, copyOffset, "\":\"", 3 );
-    APPEND_STRING( pJobsRequest, copyOffset, pStatus, statusLength );
-    APPEND_STRING( pJobsRequest, copyOffset, "\",\"", 3 );
-
-    /* Add status details if present. */
-    if( pUpdateInfo->pStatusDetails != AWS_IOT_JOBS_NO_STATUS_DETAILS )
+    else
     {
-        copyOffset = _appendStatusDetails( pJobsRequest,
-                                           copyOffset,
-                                           pUpdateInfo->pStatusDetails,
-                                           pUpdateInfo->statusDetailsLength );
-    }
+        /* Clear the request JSON. */
+        ( void ) memset( pJobsRequest, 0x00, requestLength );
 
-    /* Add expected version. */
-    if( pUpdateInfo->expectedVersion != AWS_IOT_JOBS_NO_VERSION )
-    {
-        APPEND_STRING( pJobsRequest,
-                       copyOffset,
-                       EXPECTED_VERSION_KEY,
-                       EXPECTED_VERSION_KEY_LENGTH );
+        /* Construct the request JSON. */
+        APPEND_STRING( pJobsRequest, copyOffset, "{\"", 2 );
+
+        /* Add the status. */
+        APPEND_STRING( pJobsRequest, copyOffset, STATUS_KEY, STATUS_KEY_LENGTH );
         APPEND_STRING( pJobsRequest, copyOffset, "\":\"", 3 );
-        APPEND_STRING( pJobsRequest, copyOffset, pExpectedVersion, expectedVersionLength );
+        APPEND_STRING( pJobsRequest, copyOffset, pStatus, statusLength );
```

Comment:
This change, and those below, appear to be above and beyond the scope of just removing goto statements. Should these be a separate PR with a title and description that better captures the rationale?

Reply:
I didn't change it actually. That line is same as the line 913 in original file. 
It is just the diff view problem. If you click "hide white space changes" , it will be clearer.

Reply:
Yes, it is due to uncrustify, or code shifting due to additional if/else block.